Kinvey/Kinvey/HttpRequest.swift

```diff
@@ -307,9 +313,7 @@ internal class HttpRequest: TaskProgressRequest, Request {
         self.endpoint = Endpoint.url(url: request.url!)
         self.client = client
         
-        if let authorization = request.value(forHTTPHeaderField: HttpHeader.authorization(credential: nil).name) {
-            self.credential = HttpHeaderCredential(authorization)
-        } else {
+        if request.value(forHTTPHeaderField: HttpHeaderKey.authorization.rawValue) == nil {
```

Comment:
if it's not `nil` it's because you already have the `Authorization` header, which means that we don't need to set the `credential` for this request object.